apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -24,7 +24,7 @@ import { ProfileUtils } from "@wso2is/core/utils";
 import { Field, Forms, Validation } from "@wso2is/forms";
 import { EditAvatarModal, Hint, LinkButton, PrimaryButton, UserAvatar } from "@wso2is/react-components";
 import { FormValidation } from "@wso2is/validation";
-import { isEmpty } from "lodash";
+import _, { isEmpty } from "lodash";
```

Comment:
I think we can straight away use `isEmpty` here? And `import isEmpty from "lodash/isEmpty";` will reduce the bundle size if I am not wrong.